homeassistant/components/hue.py

```diff
@@ -257,3 +265,104 @@ def set_light(self, light_id, command):
     def set_group(self, light_id, command):
         """Change light settings for a group. See phue for detail."""
         return self.bridge.set_group(light_id, command)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class HueFlowHandler(config_entries.ConfigFlowHandler):
+    """Handle a Hue config flow."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize the Hue flow."""
+        self.host = None
+
+    @property
+    def _websession(self):
+        """Return a websession.
+
+        Cannot assign in init because hass variable is not set yet.
```

Comment:
Not having access to `hass` and not passing it explicitly will bite us continuously in code review and as bugs, I think. But maybe there's no way around it?

Reply:
I guess we can make a default constructor

Reply:
One of the reasons I don't like constructors is that if people override it, we can never change that part anymore.